spec/support/shared_examples/abilities/agency_examples.rb

```diff
@@ -16,6 +16,34 @@
     should be_able_to(:review_convictions, WasteCarriersEngine::RenewingRegistration)
   end
 
+  context ":write_off_small" do
```

Comment:
I think we may be missing some tests for abilities and user types – agency users only have tests for `:write_off_small`, but not `:write_off_large`. Meanwhile some finance users have tests for `:write_off_large` (although I think base finance users have none) and none of them have tests for `:write_off_small`.

Reply:
Finance admin and finance super can do write off large, only finance-user cannot. Hence I think that's correct :) Will add more anyway! I did wrote these tests before I decided it was a good idea to split them off as suggested by Alan :D

Reply:
The users that can do things have tests that they can, but there are no tests that users are not able to do things they don't have permission to do. If that makes sense!